config/cmake/H5pubconf.h.in

```diff
@@ -538,9 +538,6 @@
  #define H5_SIZEOF_LONG_LONG 8
 #endif
 
-/* The size of `off64_t', as computed by sizeof. */
-#cmakedefine H5_SIZEOF_OFF64_T @H5_SIZEOF_OFF64_T@
```

Comment:
Though no one's likely to be using this, it still seems like it's a breaking change when we remove things from H5pubconf for a release. Not sure?